tasks_week3_Israel_Jacob/src/Zoo.cpp

```diff
@@ -0,0 +1,168 @@
+#include "Zoo.h"
+
+Zoo::~Zoo()
+{
+	for (size_t i = 0; i < list.size(); i++)
+		delete list[i];
+}
+
```

Comment:
function run is to complex and  handles multiple responsibilities such as displaying the zoo map, printing animal details, processing user commands. consider of using a class for the map and split the function into smaller function

Reply:
it is a dtor. I do not see any complexity